packages/instantsearch.js/src/lib/__tests__/InstantSearch-integration-test.ts

```diff
@@ -489,6 +489,7 @@ describe('network requests', () => {
             "params": {
               "clickAnalytics": true,
               "query": "",
+              "userToken": "cookie-key",
```

Comment:
if a cookie is present, it will always be used if the token is not no other way

Reply:
can you clarify what you mean? not sure I understand your comment